app/src/main/java/org/covidwatch/android/CovidWatchApplication.kt

```diff
@@ -90,7 +90,12 @@ class CovidWatchApplication : Application() {
             getString(R.string.preference_is_contact_event_logging_enabled),
             false
         )
-        mainScope.launch { configureAdvertising(isContactEventLoggingEnabled)  }
+        try {
+            mainScope.launch { configureAdvertising(isContactEventLoggingEnabled)  }
+        } catch (e: IllegalStateException){
+            Log.e(this.javaClass::getSimpleName.toString(),"Attempted to configure advertising while backgrounded",e)
+        }
```

Comment:
I'm not sure if this needs to be launched in mainScope, because Application.onCreate() is already running in main thread.

Reply:
Removed.  Previously it was crashing on this line number complaining that Services must started on the main thread.  Looks like an unrelated change has fixed it, since it no longer crashes now that I've removed the the coroutine scope.